build/release/pre_cloudbuild.yaml

```diff
@@ -13,16 +13,24 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 steps:
+
+  #
+  # Clean out the working dir, so we can replace it.
+  #
+  - name: ubuntu
+    script: rm *
```

Comment:
Could we include `-r` here?

Reply:
Don't need to - there's only 2 yaml files in there.